### ISSUE TITLE
itlwmx not working with Black80211

### ISSUE THREAD

**Issue by naveenkrdy** [usr-sse2/Black80211-Catalina#12]
Hi !
I wanted to try it as i have a AX200 card and itlwmx drives it. But after checking the source commits on your itlwm fork. it seems like Black80211 is yet to be integrated into itlwmx ?

Thanks !

**Comment by usr-sse2** [comment]
Yes, I concentrated my efforts on itlwm because I don't have AX cards. I'd like to fix some remaining critical issues in itlwm and probably merge to upstream before blindly porting to itlwmx.

**Comment by naveenkrdy** [comment]
@usr-sse2 sounds good : )
will be waiting for the itlwmx support. let me know if any help is needed with testing.

Thanks for your work !

**Comment by borygo77** [comment]
Will come back here soon for itlwmx support, starred! :) Well done Michael! :)
